src/main/java/uk/gov/pay/api/app/config/RateLimiterConfig.java

```diff
@@ -27,6 +28,7 @@
     @Min(1)
     private int noOfReqForPostPerNode;
 
+    @Valid
```

Comment:
We need more than this. See https://github.com/alphagov/pay-connector/blob/86317f2faea6c1024d61fc4f7e3005f63d50e2fe/src/main/java/uk/gov/pay/connector/app/SqsConfig.java#L9 for example

Reply:
Not needed. Each `noOf...` has `@Min(1)` validation which will cause the app to fail during the startup if the value is not provided (or 0).